types/materialize-css/timepicker.d.ts

```diff
@@ -17,6 +19,18 @@ declare namespace M {
          */
         static init(els: MElements, options?: Partial<TimepickerOptions>): Timepicker[];
 
+        /**
+         * If the time is AM or PM on twelve-hour clock
+         * @default 'PM'
+         */
+        amOrPm: string;
```

Comment:
You should be able to just define this as a `"PM" | "AM"` type

Reply:
sure, I am not sure why I chose a wider type here, it's not like we are going to get another time period anytime soon :)

Reply:
Haha, lets hope not! ;)